.travis.yml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  include:
+    - language: python
+      python:
+      - "3.6"
+      install:
+        - pip3 install -r requirements3.txt
+      script:
+        - make test
+        - make lint
+        - make mypy
```

Comment:
I don't think "amundsenfrontendlibrary" Make file has mypy target. You may need to add entry like this.
https://github.com/lyft/amundsensearchlibrary/blob/master/Makefile#L16